service/src/main/java/bio/terra/catalog/schema.json

```diff
@@ -86,6 +86,10 @@
     "contributors": {
       "type": "array",
       "items": { "type": "string" }
+    },
+    "brands": {
```

Comment:
Talking to @pshapiro4broad, we may want to call this `consortiums` or something? Just thinking about it, a brand is a UI concept, but a consortium is actually what RADx/RAREx/etc is.

Reply:
Yes, I don't think `brand` is correct for dataset metadata since a brand may include data from a mix of different sources, and the dataset itself shouldn't need to know about which brands corresponds to which source. My view on things:

- A dataset has a value that describes the data source, this is "consortium" or "study" or whatever value makes sense. I think the existing "consortium" field should be used unless we can think of a reason why it won't work.
- Brand is a UI concept which may correspond to a set of filters on the data shown. A brand will need to know which consortiums (or studies) it can display.

Reply:
For the record, what the UI shows as `Consortium` is computed from the model by looking at the `dct:identifier` property of the first element of the array property `TerraDCAT_ap:hasDataCollection`. So my suggestion is to store the RADx study  in `TerraDCAT_ap:hasDataCollection`. (The code could use `dct:title` instead of `dct:identifier` if this makes more sense; right now no data in production is using this field.)